lib/spanner/config/syntax_validator.ex

```diff
@@ -54,6 +54,8 @@ defmodule Spanner.Config.SyntaxValidator do
     |> prepare_return
   end
 
+  defp validate_rule_parsing(nil),
```

Comment:
We get `nil` back from `config["rules"]`. I could use `Map.get` instead and then just drop this function head.